xml/installation-ardana-manila.xml

```diff
@@ -0,0 +1,266 @@
+   </step>
+   <step>
+    <para>
+     Mount the share on a Compute instance
+    </para>
+    <screen>&prompt.ardana;mkdir ~/test_directory
+&prompt.sudo;mount -vt nfs <replaceable>EXT-NET_ID</replaceable>:/shares/<replaceable>SHARE-SHARE-ID</replaceable> ~/test_folder</screen>
+   </step>
+  </procedure>
+  <para>
+   Mode 2: The back-end is <literal>NetApp</literal>,
+   <literal>driver_handles_share_servers = True</literal> (DHSS is
+   enabled).
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Modify the configuration
+    </para>
+    <screen>&prompt.ardana;vi /etc/manila/manila.conf.d/100-manila.conf</screen>
+    <para>
+     Add a <literal>backendNetApp</literal> section
+    </para>
+    <screen>share_driver = manila.share.drivers.netapp.common.NetAppDriver
+driver_handles_share_servers = True
+share_backend_name=backendNetApp
+netapp_login=<replaceable>NetApp_USERNAME</replaceable>
+netapp_password=<replaceable>NetApp_PASSWORD</replaceable>
+netapp_server_hostname=<replaceable>NETAPP_HOSTNAME</replaceable>
+netapp_root_volume_aggregate=<replaceable>AGGREGATE_NAME</replaceable></screen>
+    <para>
+     Add to [DEFAULT] section
+    </para>
+    <screen>enabled_share_backends = backendNetApp
+default_share_type = default1</screen>
+   </step>
+   <step>
+    <para>
+     Create a &o_sharefs; share image and verify it
+    </para>
+    <screen>&prompt.ardana;wget http://tarballs.openstack.org/manila-image-elements/images/manila-service-image-master.qcow2
+&prompt.ardana;. service.osrc;glance image-create --name <literal>manila-service-image-new</literal> \
+--file manila-service-image-master.qcow2 --disk-format qcow2 \
+--container-format bare --visibility public --progress
+&prompt.ardana;openstack image list (verify a &o_sharefs; image)</screen>
+   </step>
+   <step>
+    <para>
+     Create a share type. OpenStack docs has <link
+     xlink:href="https://docs.openstack.org/manila/pike/install/post-install.html">detailed
+     instructions</link>. Use the instructions for <literal>manila type-create
+     default_share_type True</literal> .
+    </para>
+   </step>
+   <step>
+    <para>
+     Restart services
+    </para>
+    <screen>&prompt.ardana;systemctl restart openstack-manila-api openstack-manila-share \openstack-manila-scheduler
+&prompt.ardana;manila service-list (verify services are up)</screen>
+   </step>
+   <step>
+    <para>
+     Continue creating a share. <literal>OCTAVIA-MGMT-NET</literal> can be used as <replaceable>PRIVATE_NETWORK</replaceable> in this example.
+    </para>
+    <screen>&prompt.ardana;manila share-network-create --name demo-share-network1 \
+--neutron-net-id PRIVATE_NETWORK_ID --neutron-subnet-id PRIVATE_NETWORK_SUBNET_ID
+&prompt.ardana;manila create NFS 1 --name share2 --share-network demo-share-network1</screen>
+   </step>
+  </procedure>
+ </section>
+ <section>
+  <title>Troubleshooting</title>
```

Comment:
Just a reminder that we haven't had a successful demo of Manila with **_NetApp_** backend. Thus the [Troubleshooting] section might not be permanent and will undergo changes.

Reply:
change made. Comment added about NetApp